index.html

```diff
@@ -14,22 +14,22 @@
         <div class="bpm-display">
           <span class="tempo">120</span>
           <span class="bpm">BPM</span>
+          <div class="tempo-text">Very slow</div>
```

Comment:
**Comment:** I'm just moving things around a little. I want the descriptions to be within the same sections as what they're describing.

Unrelated, the `<head>` and front matter is very well done.